ksqldb-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializer.java

```diff
@@ -131,7 +131,7 @@ private static Parser decimalParser(final SqlType sqlType) {
   }
 
   private static Parser timeParser(final SqlType sqlType) {
-    return v -> new Time(Long.parseLong(v));
+    return v -> new Time(Integer.parseInt(v));
```

Comment:
Same question about overflow if the type has a value higher than the maximum INT.

Reply:
Added a check for this